scripts/commands.js

```diff
@@ -38,6 +39,14 @@ program
   .command('versions')
   .action(versions)
 
+program
+  .command('header_deps')
```

Comment:
nit: depcheck or checkdeps maybe?

Reply:
`audit-deps` and `check-deps` was confusing. That's why I switched from `check-deps` to `header-deps` to be more precise. Let me know if you still think `check-deps` sounds better?

Reply:
the docs say `GN's include header checker` so actually maybe `check-headers`?

Reply:
maybe `gn-check-deps` or `gn-check`? :) would be pretty self-describing